frontend/server/src/DAO/Notifications.php

```diff
@@ -18,6 +18,7 @@ class Notifications extends \OmegaUp\DAO\Base\Notifications {
     const COURSE_REGISTRATION_REJECTED = 'course-registration-rejected';
     const COURSE_REGISTRATION_REQUEST = 'course-registration-request';
     const COURSE_REGISTRATION_MANUAL = 'course-registration-manual';
+    const COURSE_ADMINISTRATOR_MANUAL = 'course-administrator-manual';
```

Comment:
```suggestion
    const COURSE_ADMINISTRATOR_ADDED = 'course-administrator-added';
```

"registro manual" tiene sentido, pero "administrador manual" no es claro a lo que se refiere.